podpac/core/compositor/compositor.py

```diff
@@ -48,6 +50,7 @@ class BaseCompositor(Node):
 
     sources = tl.List(trait=NodeTrait()).tag(attr=True)
     source_coordinates = tl.Instance(Coordinates, allow_none=True, default_value=None).tag(attr=True)
+    eval_serial = tl.Bool(False)
```

Comment:
I'd love to name this `multithreading`, like the setting. It should use the value from the setting by default, e.g.

```python3
multithreading = tl.Bool()

tl.default('multithreading')
def _default_multithreading(self):
    return settings['MULTITHREADING']
```

(And the OrderedCompositor can still use a different default value). Then you can just use `if self.multithreading` below.

This is related to #451, where it would be great if *all* evaluation controls like this could be overriden by any Node. If we use the same name for this trait now, we don't have to change it later.

Reply:
You always do a much better job with names than I do. Certainly I'll make that change.